apps/schools/domains/student/components/studentList/index.tsx

```diff
@@ -12,73 +12,27 @@ import { useGetAllStudentInvitationsQuery, useGetAllStudentsQuery } from '@domai
 import EmptyWrapper from '@domains/common/components/containers/EmptyWrapper'
 import { AppRoutes, RoutePath } from '@domains/common/constants/routerEnums'
-import { defaultPaginationTablePage, defaultPaginationTablePageSize } from '@domains/common/constants/Table'
-import { scrollToTop } from '@domains/common/utils/scrollInDirection'
-import { handlePaginationChange } from '@domains/common/handlers/paginationChange'
-import { calculateResults } from '@domains/student/handlers/resultsCalculate'
-import { getTotalPages } from '@domains/common/utils/getTotalPages'
 
 export function StudentList() {
-    const [queryPaginationParams, setQueryPaginationParams] = useState({
-        invites: {
-            page: defaultPaginationTablePage,
-            pageSize: defaultPaginationTablePageSize,
-        },
-        students: {
-            page: defaultPaginationTablePage,
-            pageSize: defaultPaginationTablePageSize,
-        },
-    })
-
     const [searchRequestText, setSearchRequestText] = useState('')
     const { organizationId } = useOrganization()
 
     const { data: invites, isLoading: isLoadingInvites } = useGetAllStudentInvitationsQuery({
         circle__organization__id: organizationId,
         status: StatusesEnum.SENT,
         or_search: createSearchTextForRequest(searchRequestText, searchInvitesColumns),
-        page: queryPaginationParams.invites.page,
-        page_size: queryPaginationParams.invites.pageSize,
     })
 
     const { data: students, isFetching: isFetchingStudents } = useGetAllStudentsQuery({
         circle__organization: organizationId,
         or_search: createSearchTextForRequest(searchRequestText, searchStudentsColumns),
-        page: queryPaginationParams.students.page,
-        page_size: queryPaginationParams.students.pageSize,
-    })
-
-    const [paginationParams, setPaginationParams] = useState({
-        page: defaultPaginationTablePage,
-        pageSize: defaultPaginationTablePageSize,
     })
 
-    const resultsCalculate = useCallback(
-        () => calculateResults(paginationParams, { invites, students }),
-        [paginationParams, invites, students],
-    )
-
     const data = {
         count: (invites?.count ?? 0) + (students?.count ?? 0),
         next: invites?.next ?? '',
         previous: invites?.previous ?? '',
-        results: resultsCalculate(),
-    }
-
-    const handlePageChange = (newPage: number, newPageSize: number) => {
-        handlePaginationChange(
-            setPaginationParams,
-            setQueryPaginationParams as any,
-            {
-                invites: invites?.count,
-                students: students?.count,
-            },
-            newPage,
-            newPageSize,
-            defaultPaginationTablePage,
-            defaultPaginationTablePageSize,
-            scrollToTop,
-        )
+        results: [...((invites?.results ?? []) as unknown as TableType[]), ...(students?.results ?? [])],
```

Comment:
I did this because it's easier, we have an invitation request, and lazy invitation request. The lazy request uses the full set of data, not reformatted, so I made a type cast here